test/framework/src/main/java/org/elasticsearch/test/rest/ESRestTestCase.java

```diff
@@ -1839,6 +1839,7 @@ protected static boolean isXPackTemplate(String name) {
             case "logstash-index-template":
             case "security-index-template":
             case "data-streams-mappings":
+            case "ecs@dynamic-mappings":
```

Comment:
nit: making it clear that this component template only contains `dynamic_templates`

```suggestion
            case "ecs@dynamic-templates":
```

Reply:
Trying to answer this with the potential convention. There is a common ending `-mappings`, `-settings` that can be used to differentiate between templates for settings and mappings. This one is purely dynamic_templates which is mappings, which means it would result in `ecs@dynamic_templates-mappings`? Important is the `_`. The shorter option is  `ecs@dynamic_templates` as with ECS, it is always mappings?

Reply:
So `ecs@dynamic_templates` it is?